### PR TITLE
Remove binary resource datafiles in AllDatatypes tests

### PR DESCRIPTION
**Pull request by maciejgrzybek**
Hive is used to generate ground truth data instead of checking in binary data files into repo.

This limits the burden of conflicts on binary files whenever some changes happen to data types supported by Hive.

---

TD internal review: https://github.com/Teradata/presto/pull/352

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java [hunk 1]
```diff
@@ -94,8 +101,26 @@ private static HiveTableDefinition allHiveSimpleTypesParquetTableDefinition()
...
                         ") " +
-                        "STORED AS PARQUET")
-                .setDataSource(dataSource)
-                .build();
+                        "STORED AS PARQUET");
+    }
+
+    private static HiveDataSource getTextFileDataSource()
+    {
+        return createResourceDataSource(format(tableNameFormat, "textfile"), String.valueOf(ThreadLocalRandom.current().nextLong(Long.MAX_VALUE)), "com/facebook/presto/tests/hive/data/all_types/data.textfile");
```
Comment (electrum) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: This is a bit long and repetitive. Can you add a helper method?

Comment (maciejgrzybek) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: In next commit (0c07572e83e33b4ba1757a25a8aded5b1b37750b) this method is used only once.
I simply extracted refactor (creating those repetitions) to make the actual commit with logical change more succinct and easier to follow.
If you don't like it maybe the best would be to squash 2866c60cd7d2676e123b6fa0b614c9fece0aec72 onto 0c07572e83e33b4ba1757a25a8aded5b1b37750b?

Comment (electrum) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java]: Oh, I see, GitHub showed the commits out of order. This is fine.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**